about.html

```diff
@@ -0,0 +1,109 @@
+      <a class="nav2Item" href="./index.html">Home</a>
+      <a class="nav2Item" href="./index.html">Our Packages</a>
+      <a class="nav2Item" href="#join">Join</a>
+      <a class="nav2Item" href="#sponsor">Sponsor</a>
+      <a class="nav2Item" href="#news">News</a>
+      <a class="mainItem" href="./index.html">Contributions</a> 
+    </nav>   
+</header>
+
+<header id="mobile-header">
+  <nav class="mobile-nav"> 
+      <ul onclick="menuPopup()" id="navList" title="navigation-list">
+        <li><a class="navItem" href="./index.html">Home</a></li>
+        <li><a class="navItem" href="./index.html">Our Packages</a></li>
+        <li><a class="navItem" href="#join">Join</a></li>
+        <li><a class="navItem" href="#sponsor">Sponsor</a></li>
+        <li><a class="navItem" href="#news">News</a></li>
+        <li><a class="navItem navItem-main" href="#about">Teen Creators</a></li>
+      </ul> 
+  </nav>   
+  <nav>
+    <div onclick="menuPopup()" id="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div> 
+    <div onclick="menuPopup()" id="closeButton"> &times; </div>
+  </nav>
+</header>
+    
+    <section class="mobilemain">
+      <article class="about-page" title="about">
+          <div class="main-content">
+            <h1 class="intro-title2 aboutintro2">Teen Creators <br> Global Conference 2024</h1>
+              <p class="about-paragraph" alt="explanation-paragraph">
+                Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste aliquam eaque, quae aspernatur accusamus voluptates repudiandae nemo vero praesentium quisquam placeat quos. Quis fugit illum commodi quasi, odit doloremque qui amet, deserunt ex minus labore. Ipsa voluptates expedita illum consectetur tempore veritatis ad? Asperiores quia magni in obcaecati suscipit eos neque quisquam corrupti necessitatibus. Labore quae molestiae totam. Incidunt, autem minima facere provident est earum?
+              </p>
+              <p class="contact-us">
+                Please contact us per Email for further details about Fan Expo San Francisco 2024!
+              </p>
+              <a class="mail" href="#" class="email">teencreatorconf@youtube.com</a>
+          </div>
+        </article>
+  
+        <article class="conference-info" id="news" title="conference-news">
+            <h3 class="inner-title">Teen Creators <br> 2024 Conference</h3>
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Mollitia, est.</p>
+            <img class="about-logo" src="./sources/logo.png" alt="logo">
+        </article>   
+          
+        <article class="prev-conf" title="past-conferences">
+            <h3 class="inner-title2">View Past Conferences</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero, quam.</p>
+            
+            <section class="past-conf" id="join">
```

Comment:
[OPTIONAL] Unfortunately, previous reviewer didn't notice that this part is broken, so I will ask it as optional. Please check the overlapping footer and two cards taking out of their area and white space on the bottom.
<img width="1440" alt="Screenshot 2023-03-17 at 9 36 33 PM" src="https://user-images.githubusercontent.com/25887253/225990747-51d67376-09a3-48d0-8f33-dcd810acc3c2.png">